vita3k/interface.cpp

```diff
@@ -619,7 +619,7 @@ bool handle_events(EmuEnvState &emuenv, GuiState &gui) {
             return false;
 
         case SDL_KEYDOWN: {
-            if (ImGui::GetIO().WantTextInput || gui.is_key_locked)
+            if (ImGui::GetIO().WantTextInput || gui.is_key_locked || emuenv.ctrl.drop_inputs)
```

Comment:
Moving this after 663 line

Reply:
No. If we are dropping inputs, there only needs to be one check at the very beginning of the case, and if the check succeeds, we should continue. there is no point in even reading the input. If you want me to move the `ImGui::GetIO().WantTextInput || gui.is_key_locked` part down, I can, but the ctrl.drop_inputs should stay at the top, both for readability and practicality.

Reply:
Just done what i said, is togaly good, and remove your false in other file, have no sense

Reply:
No, your suggestion has no sense. There is absolutely ZERO reason to read the input if we are dropping the inputs. they ONLY need to be un-dropped if we are in the binds loop.

Reply:
Is inversed here, is you current code have no sense... 
Juste donné what i said

Reply:
You set your bool on false, for can read gui captured key, jyst donne what i said, onoy that, can also move code on more top for read captured key

Reply:
I thinks you wrong view, you no read any input with moving this full if contrition in 664 line
You just only allow édit keyboard input

Reply:
![image](https://github.com/Vita3K/Vita3K/assets/39563889/7f8443ce-f207-44ef-8bca-fdd6ba02615f)

Reply:
You read your code ? 
Just move this  if with continue before if (allow...
And fine

Reply:
I just drew you a whole diagram. `if(gui.is_capturing_keys...` does NOT need executed if we are dropping inputs. It ONLY needs executed when we click on the button to remap, which is the point of the false statement in the other file.

Reply:
drop input is for drop action of controller, no for block remap

Reply:
No, it's for dropping action of both keyboard and controller so that you CANNOT navigate the UI while the rebinds menu is open.

Reply:
yes i know ? and is what i explain you ? 
what i said why move code after if gui.captured is full normal